### PR TITLE
Add stats table for dashboard projects

### LINKED ISSUE #323: [Dashboard Statistic] kw/h table entry
Implement the part of the table that has kw/hr. I think that the project should store that, but if it doesn't, add it! new mockup

### LINKED ISSUE #324: [Dashboard Statistic] lbs of carbon emissions saved per month table entry
Implemennt the part that says "Carbon emissions saved" and add that it's per month. Will have to add functionality to the Project model that knows how to calculate this. new mockup

### LINKED ISSUE #325: [Dashboard Statistic] acres of trees saved table entry
Implement that row in the table: new mockup

### LINKED ISSUE #326: [Dashboard Statistic] automobile miles table entry
Add a table entry (after acres of trees saved) that notes how many automobile miles per month is equivalent to the power that this project is producing.

new mockup

### LINKED ISSUE #328: [Dashboard Statistic] dollars/month table entry
Implement the table row which says "dollars/month". Note that this is dollars SAVED on energy costs per month, not the amount repaid by the project per month. The source we use to calculate how much energy is saved will be an "on average" statistic, and the source we use to calculate it is up to whoever tackles this issue.

new mockup

### PR DESCRIPTION
**Pull request by noahsark769**
Fixes #323, fixes #324, fixes #325, fixes #326, fixes #328 

Dashboard stats table for per-project stats! @sameerav can you review?

screen shot 2015-06-22 at 7 47 16 pm
screen shot 2015-06-22 at 7 47 07 pm

### CODE REVIEW

File: revolv/templates/base/partials/dashboard_project.html [hunk 1]
```diff
@@ -57,6 +57,30 @@ <h2 class="small-10 small-offset-1 columns">{{project.tagline}}</h2>
                 <div class="days-remaining-label">day{{project.days_left|pluralize}} left</div>
             </div>
         </div>
+        <div class="row project-stats">
+            <div class="project-stats-table-container small-12 large-10 large-offset-1 columns">
+                <div class="row project-stats-table-row">
+                    <div class="small-8 medium-6 columns">Kilowatt power output</div>
```
Comment (sameerav) [inline revolv/templates/base/partials/dashboard_project.html]: Do we want to make these headings cms-able? In case there's a better description

**Comment by sameerav** [comment]
lgtm to me but:

1. 
screen shot 2015-06-22 at 10 38 28 pm

is there a reason for the selectable behavior?

**Comment by noahsark769** [comment]
Oh that's weird, I'll take the selectableness out. And yeah I realized
after submitting this that I should really make those editable. Will do
that!

**Comment by noahsark769** [comment]
@sameerav how is that? I made the descriptions CMS editable but I can't reproduce the selection thing on either chrome or firefox

**Comment by sameerav** [comment]
huh maybe the selectableness is just my computer then
looks good to me, ship it! :)

**Comment by noahsark769** [comment]
Sweet, get ready for a deluge of slack notifications

**Pull request merged**